template/runtime/src/lib.rs

```diff
@@ -221,7 +221,7 @@ parameter_types! {
 impl pallet_timestamp::Config for Runtime {
 	/// A timestamp: milliseconds since the unix epoch.
 	type Moment = u64;
-	type OnTimestampSet = Aura;
+	type OnTimestampSet = ();
```

Comment:
This is going to prevent Aura's handler from doing its check:

https://github.com/paritytech/substrate/blob/ca63242fc848b9bff739c246e07310dd15ec25eb/frame/aura/src/lib.rs#L245

Thoughts on what we want to do for the template runtime?

Reply:
@sorpaas

Reply:
Please create an issue, and add a comment this line with the issue link.

Let's leave it as it is right now. We need to do a refactoring for the handling of aura and manual seal dispatch.

Reply:
Or if you have time -- the refactoring is simple as well. Just put all manual seal logic under a feature gate, so that people can enable that feature at build time to use manual seal (rather than selecting it at runtime).

Reply:
Sure!

Reply:
I ended up opening issue #315. I tried a couple different approaches to using a feature flag to solve this; I can elaborate on this later. If you're ok with that for now, @sorpaas , I think this is ready for your review.

Reply:
Let's merge this first then. Please add the issue number as comment to this line!

Reply:
Done -- see 0c2147d

Reply:
Still needs to resolve conflicts!